isofit/radiative_transfer/sRTMnet.py

```diff
@@ -215,12 +215,8 @@ def build_sixs_config(engine_config):
         + datetime.timedelta(hours=data["GEOMETRY"]["GMTIME"])
     )
 
-    solar_azimuth, solar_zenith, ra, dec, h = sunpos(
-        dt,
-        data["GEOMETRY"]["PARM1"],
-        -data["GEOMETRY"]["PARM2"],
-        data["SURFACE"]["GNDALT"] * 1000.0,
-    )
+    solar_azimuth = data["GEOMETRY"]["PARM1"]
+    solar_zenith = data["GEOMETRY"]["PARM2"]
```

Comment:
@pgbrodrick I removed the calculation of the sun position here as both solar azimuth and zenith angles are now directly provided by the updated MODTRAN template (using IPARM 12).